packages/remix-react/components.tsx

```diff
@@ -1281,14 +1284,39 @@ export function LiveReload({ port = 8002 }: { port?: number }) {
               console.log("💿 Reloading window ...");
               window.location.reload();
             }
+            if(event.type === "REBUILDING") {
+              spinner.classList.add("loading");
+            }
           };
           ws.onerror = error => {
             console.log("Remix dev asset server web socket error:");
             console.error(error);
           };
       `
-      }}
-    />
+        }}
+      />
+      <style
+        dangerouslySetInnerHTML={{
+          __html: `
+        #__remix-livereload-spinner {
+          visibility: hidden;
+          position: fixed;
+          bottom: 10px;
+          right: 10px;
+          font-size: 20px;
+          user-select: none;
+        }
+        #__remix-livereload-spinner.loading {
+          visibility: visible;
+          animation: spin 1s linear infinite;
+        }
+        @keyframes spin {
```

Comment:
I'd recommend adding a prefix to the animation name to prevent any possible conflicts.

Reply:
good point

Reply:
I checked and it has really good support https://caniuse.com/mdn-css_at-rules_keyframes also searched the repo found multiple uses of it without prefix I believe it's good to leave it without one

Reply:
Sorry I mean prefix the name of the animation. Incase someone using Remix has another animation called spin.

Reply:
aha got it!

Reply:
addressed in https://github.com/remix-run/remix/pull/664/commits/23a0df466a6ed23f515a4df2afdd173563e8562c will probably squash all before merging